docs/spark/how-to-guides/connect-to-azure-storage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Connect to Remote Storage from your local machine
+description: Connect to Azure Storage Accounts using .NET for Apache Spark from your local machine.
+ms.date: 09/24/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Connect to Azure Data Lake Storage Gen 2 or WASB account
+
+In this article, you learn how to connect to an ADLS (Azure Data Lake Storage) Gen 2 or WASB (Windows Azure Storage Blob) account through an instance of [.NET for Apache Spark](https://github.com/dotnet/spark) running locally on your Windows machine.
+
+## Steps to setup the environment
+
+1. Download the Apache Spark distribution built without Hadoop from [official website](https://archive.apache.org/dist/spark/) (choose a version [supported by .NET for Apache Spark](https://github.com/dotnet/spark#supported-apache-spark)), and extract it to a directory. Set the environment variable `SPARK_HOME` to this directory.
+2. Download the Apache Hadoop binary from [here](http://hadoop.apache.org/releases.html) and extract to a folder, and set your `HADOOP_HOME` environment variable to this folder.
+3. Download the `winutils.exe` and `hadoop.dll` files from [this location](https://github.com/cdarlint/winutils) (depending on the version of Hadoop installed in the previous step) and put them in the bin folder of your Hadoop. These binaries are needed on Windows in order to get everything setup correctly (this is explained in detail in the [Apache wiki here](https://cwiki.apache.org/confluence/display/HADOOP2/WindowsProblems)).
+4. Configure your Hadoop installation by making the following changes to your `%HADOOP_HOME%\etc\hadoop\hadoop-env.cmd` file:
+    1. Set the `JAVA_HOME` property using the DOS path (since Hadoop doesn't like spaces in `JAVA_HOME`). This should look something like this: `C:\Progra~1\Java\jdk1.8.0_241` (Pointing to whatever version of Java you have installed on your local machine).
+    2. Append `%HADOOP_HOME%\share\hadoop\tools\lib\*` to `HADOOP_CLASSPATH`.
+    Your final `hadoop-env.cmd` file should look something like this:
+    ![Final hadoop-env.cmd file](./media/connect-external-sources/hadoop-env.png)
```

Comment:
Images that aren't surrounded by blank lines used to cause problems in the localized docs.

```suggestion

    ![Final hadoop-env.cmd file](./media/connect-external-sources/hadoop-env.png)
```